### PR TITLE
Fix #401 SSL Labs upgrade

### LINKED ISSUE #401: SSL Labs Upgrade with new API access
https://www.qualys.com/company/newsroom/news-releases/usa/2015-03-17-qualys-enhances-ssl-labs-service-new-api-access-safeguard-websites/

### PR DESCRIPTION
**Pull request by pauTE**
Hi @delta24 I have added the new api and the new call in /profiles/resources/default.cfg. And I have added a new python script in order to parse the important values(the ones you said) from the report.json. into a new html file.Everything seems to work fine. Please ,review.

### CODE REVIEW

File: scripts/ssl/sslScanParser.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    f.write('<div style="background-color: lightgrey;border:1px solid;border-radius:10px;width:300px;"')
+    f.write('<h1>SSL SCAN REPORT</h1>')
+    f.write('</div>')
+    f.write('<h2> Input Url :'+data[0]['host']+'</h2>')
+    f.write('<h2> Domain :'+data[0]['host']+'</h2>')
+    f.write('<h2' + str(data[0]['endpoints'][0]['details']['httpStatusCode'])+'</h2>')
+    f.write('<h2> IP :'+data[0]['endpoints'][0]['ipAddress']+'</h2>')
+    f.write('<h2> Grade :'+data[0]['endpoints'][0]['grade']+'</h2>')
+    f.write('<h2> Secondary Grade :'+data[0]['endpoints'][0]['gradeTrustIgnored']+'</h2>')
+    f.write('<h2> Freak : '+str(data[0]['endpoints'][0]['details']['freak'])+'</h2>')
```
Comment (delta24) [inline scripts/ssl/sslScanParser.py]: This should be `f.write('<h2> Freak :</h2> '+str(data[0]['endpoints'][0]['details']['freak']))`. This makes it more readable - the values are of different font size for `key:` and `value`.

**Comment by pauTE** [comment]
@delta24 Done. Thanks

**Comment by delta24** [comment]
One small thing now - I forgot earlier: make the filename `reporter.py` or `sslscan_parser.py` (your call).

**Comment by pauTE** [comment]
@delta24  Done ;)

**Pull request merged**